src/Joystick/Joystick.h

```diff
@@ -152,7 +152,7 @@ class Joystick : public QThread
     
     CalibrationMode_t   _calibrationMode;
     
-    static const int    _cAxes = 4;
+    static const int    _cAxes = 6;
```

Comment:
This is not the real fix. The axes need to come from the axis count detection of the joystick.